docs/source/notebooks/wps_convert_demo.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>Thank goodness. Hopefully with <code>reviewnb</code>  we don't miss these kinds of mistakes again.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/osprey/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>